src/icons/fileIcons.ts

```diff
@@ -904,7 +904,7 @@ export const fileIcons: FileIcons = {
         { name: 'capacitor', fileNames: ['capacitor.config.json'] },
         { name: 'sketch', fileExtensions: ['sketch'] },
         { name: 'pawn', fileExtensions: ['pwn', 'amx'] },
-        { name: 'adonis', fileNames: ['.adonisrc.json'] },
+        { name: 'adonis', fileNames: ['.adonisrc.json', 'ace'] },
```

Comment:
I'm just wondering if "ace" is a file name or a file extension. Could you please provide more details about that? Are there really some files called "ace" and what is their purpose?

Reply:
Yeah, that's filename without extension

Reply:
CLI helper

Reply:
Alright, thank you for clarifying.